### PR TITLE
Set file permissions explicitly

### PR DESCRIPTION
**Pull request by swalkinshaw**
Fixes ansible-lint violations

### CODE REVIEW

**Review by TangRufus — CHANGES_REQUESTED** [review]
Always use octal numbers?
```diff
- mode: '0644'
+ mode: 0644
```

File: roles/wordpress-setup/tasks/self-signed-certificate.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@
   template:
     src: self-signed-openssl-config.j2
     dest: "{{ nginx_ssl_path }}/self-signed-openssl-configs/{{ item.key }}.cnf"
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/self-signed-certificate.yml:12]:
```suggestion
    mode: 0644
```

File: roles/wordpress-setup/tasks/nginx.yml [hunk 2]
```diff
@@ -52,12 +53,14 @@
...
     src: "{{ playbook_dir }}/roles/letsencrypt/templates/acme-challenge-location.conf.j2"
     dest: "{{ nginx_path }}/acme-challenge-location.conf"
+    mode: '0644'
   notify: reload nginx
 
 - name: Create WordPress configuration for Nginx
   template:
     src: "{{ item.value.nginx_wordpress_site_conf | default(nginx_wordpress_site_conf) }}"
     dest: "{{ nginx_path }}/sites-available/{{ item.key }}.conf"
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/nginx.yml:63]:
```suggestion
    mode: 0644
```

File: roles/wordpress-setup/tasks/nginx.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@
   template:
     src: "{{ item.src }}"
     dest: "{{ nginx_path }}/sites-available/{{ item.src | basename | regex_replace('.j2$', '') }}"
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/nginx.yml:26]:
```suggestion
    mode: 0644
```

File: roles/wordpress-setup/tasks/nginx.yml [hunk 4]
```diff
@@ -52,12 +53,14 @@
   template:
     src: "{{ playbook_dir }}/roles/letsencrypt/templates/acme-challenge-location.conf.j2"
     dest: "{{ nginx_path }}/acme-challenge-location.conf"
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/nginx.yml:56]:
```suggestion
    mode: 0644
```

File: roles/wordpress-setup/tasks/nginx-includes.yml [hunk 5]
```diff
@@ -24,6 +24,7 @@
   template:
     src: "{{ item }}"
     dest: "{{ nginx_path }}/includes.d/{{ item | regex_replace(nginx_includes_pattern, '\\2') }}"
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/nginx-includes.yml:27]:
```suggestion
    mode: 0644
```

File: roles/wordpress-setup/tasks/main.yml [hunk 6]
```diff
@@ -27,6 +27,7 @@
   template:
     src: php-fpm.conf.j2
     dest: /etc/php/7.4/fpm/pool.d/wordpress.conf
+    mode: '0644'
```
Comment (TangRufus) [inline roles/wordpress-setup/tasks/main.yml:30]:
```suggestion
    mode: 0644
```

**Comment by swalkinshaw** [comment]
It works either way but converted them all for consistency

**Review by TangRufus — APPROVED** [review]

**Pull request merged**